manifests/z/Ziad/TCG/6.8/Ziad.TCG.installer.yaml

```diff
@@ -0,0 +1,28 @@
+# Created with YamlCreate.ps1 v2.2.8 $debug=NVS1.CRLF.5-1-19041-3031.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Ziad.TCG
+PackageVersion: "6.8"
+InstallerLocale: en-US
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /verysilent
+  SilentWithProgress: /silent
+UpgradeBehavior: uninstallPrevious
+Commands:
+- temp_cleaner_gui
+ReleaseDate: 2023-07-03
+Installers:
+- Architecture: x64
+  InstallerUrl: https://liquidtelecom.dl.sourceforge.net/project/temp-cleaner-gui/temp_cleaner_gui_v6.8-stable_win7sp1to11_amd64_mrx.exe
+  InstallerSha256: A4C1123196C2FE4337F4B9F89C6E8F47DDC12E0105FFE0CA7434322EFFCFF792
```

Comment:
```suggestion
  InstallerSha256: A4C1123196C2FE4337F4B9F89C6E8F47DDC12E0105FFE0CA7434322EFFCFF792
  AppsAndFeaturesEntries:
    - DisplayVersion: 6.8-stable
      ProductCode: '{27AE9354-A3A7-4588-BEBF-FD41E3EF1E46}_is1'
```

The version written to registry (visible in Control Panel / Apps&Features) is different from the version used for marketing. WinGet relies on the actual DisplayVersion written to registry to match upgrades so we need to add AppsAndFeaturesEntries node in the manifest to distinguish the two versions.

Reply:
I don't actually see the necessity of putting "stable" after the version number as I don't publish development or beta versions of Temp_Cleaner GUI to Winget.

Reply:
Actually, it's the installer that's writing this value to Registry. WinGet relies on metadata from the registry to match upgrades. This value is what installer wrote to the registry that you can also see in `appwiz.cpl` / Apps&Features. If you don't include this value, WinGet won't be able to correctly correlate the installed application to the manifest available here in the repo; That _could_ result in an infinite upgrade loop for the user.
![image](https://github.com/microsoft/winget-pkgs/assets/88161975/f1bfdb21-d2d9-4680-9b0f-f6c917901e1e)

The actual fix is to modify the installer that's writing this value to the registry and remove the `-stable` suffix.

Reply:
Thanks for clarifying that, I will make sure to include the `-stable` suffix the next time I create a manifest.